src/vue/text-editor/text-editor.vue

```diff
@@ -1,23 +1,23 @@
 <template>
   <div class="alpheios-alignment-editor-container">
-      <h2>Define Source and Translation Texts 
+      <h2>Define Origin and Target Texts 
         (<span class="alpheios-alignment-editor-text-define-container__show-label" @click="toggleDefineTextsShow">{{ defineTextsShowLabel }}</span>)
       </h2>
       <div class="alpheios-alignment-editor-text-define-container" v-show="defineTextsShow">
         <div class="alpheios-alignment-editor-text-define-container-inner">
-          <div class="alpheios-alignment-editor-text-container alpheios-alignment-editor-source-text-container">
+          <div class="alpheios-alignment-editor-text-container alpheios-alignment-editor-origin-text-container">
             <text-editor-single-block 
-                text-id="source" 
-                @update-text = "updateSourceText" 
-                :external-text = "updatedSource"
+                text-id="origin" 

```

Comment:
If it's a custom attribute, I think it's better be defined as a `data-` attribute, because otherwise it won't pass a validator check (and using `data-` prefix for custom attributes is generally a good practice). Can be applied to this and other places too

Reply:
@kirlat , I think I didn't understand your suggestion here, could you explain a little more.
 - what validator do you mean here? that is not defined. I have here a simple string validater as a vue property
 - I have no reasons to print this attribute to the html layout
 - what are the preferences for injecting some props with data and some of them without data prefix? Why it should not be unified?

Reply:
If I read the above correctly, this will result in the following HTML: `<text-editor-single-block text-id="origin" external-text = "updatedOrigin">`. If we (or someone else) do a validation of such HTML source, HTML validators will report it as incorrect. The reason would be the unknown attributes of the element. The ones that are "allowed" are, I believe, those listed on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes), and neither `text-id` nor `external-text` are not among them.

It is widely recommended to prefix custom attributes with `data-` making them the data attributes. Data attributes will not cause any issues with validation. So why wouldn't we rename them to `data-text-id` and `data-external-text`? It'll be not a big deal for us probably, but will make the pundits happy and will follow the guidelines. What do you think?

Reply:
but in this case, it's not a valid HTML element anyway, it's a component. Do the same rules apply to components?

Reply:
Oh, yes, you're right. I've missed that it's a component. Then it's all perfect as it is and all I was saying does not make sense in regards to a components. Sorry for the confusion introduced.